commit 2af13ef2dccbe7c3dc88b4f2b5a5329f0e135a99
Author: Jérôme Quelin <jquelin@mandriva.org>
Date:   Fri Mar 12 17:22:23 2010 +0000

    skip plain, regular comments. how comes this has not been spotted earlier? (sigh)

diff --git a/scripts/perl.prov b/scripts/perl.prov
index 8f5d53b..3113cb2 100755
--- a/scripts/perl.prov
+++ b/scripts/perl.prov
@@ -117,7 +117,7 @@ sub process_file {
       $inover = 0;
     }
 
-    if ($incomment || $inover) {
+    if ($incomment || $inover || m/^\s*#/) {
        next;
     }
 
